src/components/Request/NewRequestForm.tsx

```diff
@@ -0,0 +1,196 @@
+import { Label, Title1, Button } from "@fluentui/react-components";
+import { useForm } from "react-hook-form";
+import { ContactIcon } from "@fluentui/react-icons-mdl2";
+import { RPARequest } from "api/requestsApi";
+import { useCurrentUser } from "api/UserApi";
+import {
+  RequestType,
+  MCRRequired,
+  PaySystem,
+  HiringType,
+  AdvertisementLength,
+  Incumbent,
+} from "./FormFields/FormFields";
+import "./Request.css";
+
+// Probably use the omit system with RPARequest later, too simple right now
+export type RHFRequest = {
+  requestType: string;
+  mcrRequired: string;
+  paySystem: string; // "NH" | "GS" | "GG";
+  hireType: string; // "Internal" | "External";
+  advertisementLength: string; // "Normal" | "Extended";
+  lastIncumbent: string;
+};
+
+const NewRequestForm = () => {
+  const user = useCurrentUser();
+  //const addRequest = useAddRequest();
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+    setValue,
+  } = useForm<RHFRequest>({
+    defaultValues: {
+      requestType: "",
+      mcrRequired: "",
+      paySystem: "NH",
+      hireType: "",
+      advertisementLength: "",
+      lastIncumbent: "",
+    },
+    criteriaMode:
+      "all" /* Pass back multiple errors, so we can prioritize which one(s) to show */,
+    mode: "onChange" /* Provide input directly as they input, so if entering bad data (eg letter in MPCN) it will let them know */,
+  });
+
+  const createNewRequest = async (data: RHFRequest) => {
+    //TODO: properly convert from RHFRequest to Request
+    const data2 = {
+      ...data,
+    } as RPARequest;
+
+    //const newRequest = await addRequest.mutateAsync(data2);
+    console.log(data2);
+  };
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        width: "100%",
+        alignItems: "center",
+      }}
+    >
+      <Title1 align="center">
+        <b>Initiate New RPA Request</b>
+      </Title1>
+
+      <form
+        id="inReqForm"
+        className="requestFormContainer"
+        onSubmit={handleSubmit(createNewRequest)}
+      >
+        {/* Requestor */}
+        <div className="requestFieldContainer">
+          <Label
+            id="requestorId"
+            size="small"
+            weight="semibold"
+            className="requestFieldLabel"
+            required
+          >
+            <ContactIcon className="requestFieldIcon" />
+            Requestor
+          </Label>
+          {user.text}
+        </div>
+
+        <RequestType
+          name="requestType"
+          control={control}
+          errors={errors}
+          setValue={setValue}
+        />
```

Comment:
Love the splitting into separate components for each field.  May be able to simplify by not passing down props, but use FormProvider and useFormContext()